### PR TITLE
Device-related updates

### PR DESCRIPTION
**Pull request by david-cho-lerat-HL2**

### CODE REVIEW

File: openapi/spec.json [hunk 1]
```diff
@@ -1177,6 +1183,168 @@
         }
       }
     },
+    "/devices/{organizationId}/{deviceId}/routes/{routeId}/activate": {
```
Comment (kevinpollet) [inline openapi/spec.json]: What's the returned status if we activate an already activated route?

File: openapi/spec.json [hunk 2]
```diff
@@ -1177,6 +1183,168 @@
...
+          "429": {
+            "$ref": "#/components/responses/429"
+          },
+          "500": {
+            "$ref": "#/components/responses/500"
+          }
+        }
+      }
+    },
+    "/devices/{organizationId}/{deviceId}/routes/{routeId}/deactivate": {
```
Comment (kevinpollet) [inline openapi/spec.json]: What's the returned status if we deactivate an already activated route?

File: openapi/spec.json [hunk 3]
```diff
@@ -1177,6 +1183,168 @@
...
+        ],
+        "summary": "Get a provider",
+        "description": "Get the details for a provider given its ID.",
+        "responses": {
+          "200": {
+            "description": "Success",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProviderInArray"
```
Comment (kevinpollet) [inline openapi/spec.json]: If there is no difference between the ressources returned in the list endpoint, I think the schema must renamed to `Provider`.

Comment (david-cho-lerat-HL2) [reply openapi/spec.json]: ok

File: openapi/spec.json (hunk 1 above)
Comment (david-cho-lerat-HL2) [reply openapi/spec.json]: "404 Not Found"

File: openapi/spec.json (hunk 2 above)
Comment (david-cho-lerat-HL2) [reply openapi/spec.json]: "404 Not Found"

Comment (david-cho-lerat-HL2) [reply openapi/spec.json]: image
I will add this to the doc

File: openapi/spec.json (hunk 1 above)
Comment (david-cho-lerat-HL2) [reply openapi/spec.json]: image
I will add this to the doc

**Pull request merged**